src/braket/circuits/circuit.py

```diff
@@ -1495,7 +1496,7 @@ def __repr__(self) -> str:
             )
 
     def __str__(self):
-        return self.diagram(AsciiCircuitDiagram)
+        return self.diagram(self.default_diagram_builder)
```

Comment:
I think this should just call self.diagram() and use the default.

I think we should change def diagram() to use the new box drawing class as the default class.